libraries/AP_OSD/AP_OSD.h

```diff
@@ -0,0 +1,112 @@
+/*
+ * This file is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This file is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#pragma once
+
+#include <AP_Param/AP_Param.h>
+
+class AP_OSD_Backend;
+
+#define AP_OSD_NUM_SCREENS 4
+
+/*
+  class to hold one setting
+ */
+class AP_OSD_Setting {
+public:
+    AP_Int8 enabled;
+    AP_Int8 xpos;
+    AP_Int8 ypos;
+
+    AP_OSD_Setting(bool enabled, uint8_t x, uint8_t y);
+    
+    // User settable parameters
+    static const struct AP_Param::GroupInfo var_info[];
+};
+
+
+/*
+  class to hold one screen of settings
+ */
+class AP_OSD_Screen {
+public:
+    // constructor
+    AP_OSD_Screen();
+    
+    void draw(void);
+
+    void set_backend(AP_OSD_Backend *_backend) { backend = _backend; };
+
+    // User settable parameters
+    static const struct AP_Param::GroupInfo var_info[];
+
+    AP_Int8 enabled;
+    
+private:
+    AP_OSD_Backend *backend;
+
+    AP_OSD_Setting altitude{true, 1, 1};
```

Comment:
Thank you for review! Having (false, x, y) is make sense, because user could change _EN to true and get item in some resonable place. 0,0 is not visible with some screen and camera combo